src/Plugin/GraphQL/Scalars/TypedData/Untyped.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Drupal\graphql_views\Plugin\GraphQL\Scalars\TypedData;
+
+use Drupal\graphql\Plugin\GraphQL\Scalars\ScalarPluginBase;
+use GraphQL\Utils\AST;
+
+/**
+ * Input types for view contextual filters.
+ *
+ * @GraphQLScalar(
+ *   id = "untyped",
+ *   name = "Untyped",
+ *   type = "Untyped",
+ *   provider = "views"
+ * )
+ */
+class Untyped extends ScalarPluginBase {
```

Comment:
This should not be called "Untyped" though but still be called by its "purpose", e.g. "ViewsFilterInput"